### PR TITLE
Rename FindParser class name to improve readability

### PR DESCRIPTION
**Pull request by wilsonngja**

### CODE REVIEW

File: src/main/java/seedu/duke/parsermanager/ParserManager.java [hunk 1]
```diff
@@ -174,9 +175,9 @@ private Parser parseFindCommand(String commandDetail) throws UndefinedSubCommand
 
         String findCommandDescription = findCommandTypeAndFlag.get(COMMAND_DESCRIPTION_INDEX);
         if (isFindClient) {
-            return new ParseFindClient(findCommandDescription);
+            return new CommandFindClientParser(findCommandDescription);
         } else if (isFindProperty) {
-            return new ParseFindProperty(findCommandDescription);
+            return new CommandFindPropertyParser(findCommandDescription);
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/parsermanager/ParserManager.java:180]: Good naming

**Review by ngdeqi — COMMENTED** [review]
Follows the coding quality guidelines. LGTM

File: src/main/java/seedu/duke/parsermanager/ParserManager.java (hunk 1 above)
Comment (OVReader) [reply src/main/java/seedu/duke/parsermanager/ParserManager.java:180]: Agreed, nicely done!

**Review by OVReader — APPROVED** [review]
Good change from `FindParser` to `CommandFindParser` class naming which is more suitable for class names as classes should sound like nouns.

**Pull request merged**